packages/fxa-content-server/tests/functional/subscription_terms.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { registerSuite } = intern.getInterface('object');
+const FunctionalHelpers = require('./lib/helpers');
+var PAGE_URL = intern._config.fxaContentRoot + 'legal/subscription_terms';
+var LEGAL_URL = intern._config.fxaContentRoot + 'legal';
+
+const { noSuchElement } = FunctionalHelpers;
+
+registerSuite('subscription platform terms of service', {
+  beforeEach: function() {
+    return this.remote.then(FunctionalHelpers.clearBrowserState());
+  },
+  tests: {
+    'start at /legal': function() {
+      return this.remote
+        .get(LEGAL_URL)
+        .setFindTimeout(intern._config.pageLoadTimeout)
+        .findByLinkText('Subscription Platform Terms of Service')
+        .click()
+        .end()
+
+        .findByCssSelector('#fxa-subscription-terms-back')
+        .click()
```

Comment:
`click` helper can replace the 3 lines here and above

Reply:
Thanks!  I copied and edited the tests for another legal page... 😳